src/components/SubpageBoxInput.js

```diff
@@ -301,14 +323,20 @@ class SubpageBoxInput extends Component {
                                 handleSubpageBoxChange(event, subpage.id)
                             }
                         >
-                            {orderTypes.map(option => (
-                                <MenuItem key={uuidv1()} value={option.value}>
-                                    {option.name}
-                                </MenuItem>
-                            ))}
+                            {Object.keys(constantsOrderTypes.orderTypes).map(
+                                option => (
+                                    <MenuItem key={uuidv1()} value={option}>
```

Comment:
I think that you could just use `option` as it's already unique.
```suggestion
                                    <MenuItem key={option} value={option}>
```